kentikapi/client.go

```diff
@@ -228,3 +260,23 @@ func (c Config) makeAuthInterceptor() grpc.UnaryClientInterceptor {
 		return invoker(ctx, method, req, reply, cc, opts...)
 	}
 }
+
+func (c Config) makeRetryInterceptor() grpc.UnaryClientInterceptor {
+	opts := []grpc_retry.CallOption{
```

Comment:
I am pretty sure that there is no need to create local slice - you can just pass those 3 options to directly to _UnaryClientInterceptor()_ call

Reply:
Done